server/tests/api/userTests.js

```diff
@@ -0,0 +1,72 @@
+import chai from 'chai';
+import request from 'supertest';
+import 'babel-polyfill';
+
+import User from './../../models/User';
+import users from './../testData/users';
+import app from './../../app';
+
+const { expect } = chai;
+
+describe('UserController', () => {
+  before(async () => {
+    await User.remove({});
+  });
+
+  describe('When a user signs up', () => {
+    it('Then it should return the user details and authentication token', async () => {
+      const response = await request(app)
+        .post('/api/v1/users/signup')
+        .set('Accept', 'application/x-www-form-urlencoded')
+        .send(users[0])
+        .expect(201);
+
+      expect(response.body.user).to.be.an('object');
```

Comment:
I think its better to `(expect.body).to.have.property('user')`. Same thing for the remaining tests below

Reply:
Okay. Thanks. Will do that